diff --git a/ethereum/contracts/bridge/Bridge.sol b/ethereum/contracts/bridge/Bridge.sol
index e4cd0df0..6ff3d1d7 100644
--- a/ethereum/contracts/bridge/Bridge.sol
+++ b/ethereum/contracts/bridge/Bridge.sol
@@ -58,7 +58,7 @@ contract Bridge is BridgeGovernance, ReentrancyGuard {
 
         sequence = wormhole().publishMessage{
             value : msg.value
-        }(0, nonce, encoded, 15);
+        }(0, nonce, encoded, 1);
     }
 
     function wrapAndTransferETH(uint16 recipientChain, bytes32 recipient, uint256 arbiterFee, uint32 nonce) public payable returns (uint64 sequence) {
@@ -194,7 +194,7 @@ contract Bridge is BridgeGovernance, ReentrancyGuard {
 
         sequence = wormhole().publishMessage{
             value : callValue
-        }(recipientChain, nonce, encoded, 15);
+        }(recipientChain, nonce, encoded, 1);
     }
 
     function updateWrapped(bytes memory encodedVm) external returns (address token) {
diff --git a/ethereum/contracts/nft/NFTBridge.sol b/ethereum/contracts/nft/NFTBridge.sol
index 86ae37eb..ade4a34f 100644
--- a/ethereum/contracts/nft/NFTBridge.sol
+++ b/ethereum/contracts/nft/NFTBridge.sol
@@ -93,7 +93,7 @@ contract NFTBridge is NFTBridgeGovernance {
 
         sequence = wormhole().publishMessage{
             value : callValue
-        }(targetChainId, nonce, encoded, 15);
+        }(targetChainId, nonce, encoded, 1);
     }
 
     function completeTransfer(bytes memory encodedVm) public {
